.changeset/early-pigs-move.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nacelle/storefront-sdk': patch
+---
+
+Remove the storefront method as the default export for the SDK, a new instance of the `StorefrontClient` can be used instead.
```

Comment:
Because the `Storefront` function was a named export, I think we should avoid using "default export" here:

```suggestion
Removed the `Storefront` function that initialized the Storefront SDK. Instead, the SDK is initialized by creating a new instance of the `StorefrontClient` class.
```